README.md

```diff
@@ -294,3 +295,5 @@ And finally, you are ready to go! Or at least you should be...
 [homebrew]: <https://brew.sh/>
 [IPerM]: <ian.perez@alum.esci.upf.edu>
 [passphrase]: <https://docs.github.com/en/authentication/connecting-to-github-with-ssh/working-with-ssh-key-passphrases>
+[pageale]: https://github.com/pageale>
+< 
```

Comment:
I think line 299 is a typo! Please delete the less than symbol - the rest is perfect!